code/datums/gamemode/objectives/target/delayedassassinate.dm

```diff
@@ -10,7 +10,8 @@
 		if(find_target())
 			explanation_text = format_explanation()
 			if(owner)
-				to_chat(owner.current,"<span class='warning'><BIG>We have identified our objective target!</BIG></span>")
+				to_chat(owner.current,"<span class='warning'><BIG>We have identified our objective!</BIG></span>")
+				to_chat(owner.current,"<span class='warning'>The target is [target.current.real_name], the [target.assigned_role=="MODE" ? (target.special_role) : (target.assigned_role)]. Commit it to memory.</span>")
```

Comment:
"Commit that name" to memory might make more sense here, rather than "it"

Reply:
What about "It has been added to your notes" to make it obvious to players that the notes are a thing that exists in case they're not aware.

Reply:
That's what I'm trying to accomplish with the 'commit it to memory' line in the first place.